client/my-sites/post-share/style.scss

```diff
@@ -1,177 +1,278 @@
-		display: flex;
-		flex-wrap: wrap;
-		margin-top: 16px;
-		padding-bottom: 20px;
+.post-share__subtitle {
+	margin-bottom: 16px;
+}
 
-		@include breakpoint( ">480px" ) {
-		}
+.post-share__main {
+	border-top: $section-border;
+	display: flex;
+	flex-direction: column-reverse;
+
+	@include breakpoint( ">480px" ) {
+		flex-direction: row;
 	}
 
-	.post-share__services-header {
-		margin-bottom: 4px;
-		padding-left: 20px;
-		padding-bottom: 15px;
-		font-size: 14px;
-		border-bottom: $section-border;
+	@include breakpoint( ">660px" ) {
+		flex-direction: column-reverse;
 	}
 
-	.post-share__services-add {
-		text-align: center;
-		margin: 5px 20px;
+	@include breakpoint( ">960px" ) {
+		flex-direction: row;
 	}
+}
 
-	.editor-sharing__message-heading {
-		padding-left: 20px;
-		padding-bottom: 10px;
-		padding-top: 10px;
-		border-bottom: $section-border;
-		border-top: $section-border;
+.post-share__form.is-placeholder,
+.post-share__services.is-placeholder {
+	@include placeholder;
+	margin-top: 16px;
+	height: 48px;
+}
 
-		@include breakpoint( ">480px" ) {
-			padding-bottom: 15px;
-			border-top: 0;
-			padding-top: 0;
-		}
+.post-share__head {
+	padding: 0 16px;
+
+	@include breakpoint( ">480px" ) {
+		padding: 0 24px;
 	}
+}
 
-	.post-share__service {
-		cursor: pointer;
-		display: flex;
-		justify-content: left;
-		padding: 0 0 12px 4px;
-		color: lighten( $gray, 20% );
-		margin-right: 20px;
-		margin-left: 20px;
-
-		.social-logo {
-			padding-left: 4px;
+.post-share__form {
+	border-right: $section-border;
+	flex: 3;
+	padding-bottom: 16px;
+
+	.editor-sharing__publicize-message {
+			margin-top: 0;
+
+		@include breakpoint( ">480px" ) {
+			margin-top: 11px;
 		}
 
-		&:first-of-type {
-			margin-top: 8px;
+		@include breakpoint( ">660px" ) {
+			margin-top: 0;
 		}
 
-		&.is-broken {
-			cursor: not-allowed;
+		@include breakpoint( ">960px" ) {
+			margin-top: 11px;
 		}
 
-		&.is-active {
-			color: black;
+		.editor-sharing__message-heading {
+			border-top: $section-border;
+			border-bottom: $section-border;
+			line-height: 15px;
+			margin-bottom: 8px;
+			padding: 11px 16px;
 
-			.social-logo.twitter{
-				color: $color-twitter;
+			@include breakpoint( ">480px" ) {
+				border-top: 0;
+				padding: 0 24px 11px;
 			}
 
-			.social-logo.facebook{
-				color: $color-facebook;
+			@include breakpoint( ">660px" ) {
+				border-top: $section-border;
+				line-height: 28px;
+				padding-top: 11px;
 			}
 
-			.social-logo.google-plus {
-				color: $color-gplus;
+			@include breakpoint( ">960px" ) {
+				border-top: 0;
+				padding-top: 0;
 			}
+		}
 
-			.social-logo.linkedin {
-				color: $color-linkedin;
-			}
+		.editor-sharing__message-input {
+			margin: 16px 16px 8px;
 
-			.social-logo.tumblr {
-				color: $color-tumblr;
+			@include breakpoint( ">480px" ) {
+				margin-right: 24px;
+				margin-left: 24px;
 			}
+		}
 
-			.social-logo.path {
-				color: $color-path;
-			}
+		textarea.editor-sharing__message-input {
```

Comment:
Can we drop the `textarea` part?

![selection_013](https://cloud.githubusercontent.com/assets/4988512/24167522/09dd6536-0e45-11e7-99e1-f316c2de4d08.png)